src/System.Private.ServiceModel/tests/Common/Infrastructure/src/ConditionalWcfTest.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+namespace Infrastructure.Common
+{
+    public class ConditionalWcfTest
+    {
+        public static bool IsTestEnabled(string testCategoryName)
```

Comment:
NIT: I think it might read better as "IsCategoryEnabled" or "IsConditionalTestEnabled"